BankWallet/BankWallet/Modules/RateList/RateListRouter.swift

```diff
@@ -0,0 +1,28 @@
+import UIKit
+
+class RateListRouter {
+}
+
+
+extension RateListRouter: IRateListRouter {
+}
+
+extension RateListRouter {
+
+    static func module() -> UIViewController {
+        let router = RateListRouter()
+
+        let coins = RateListSorter().smartSort(for: App.shared.walletManager.wallets.map { $0.coin }, featuredCoins: App.shared.appConfigProvider.featuredCoins)
```

Comment:
All of the interactions should be made in interactor ONLY. You should not fetch any external data (coins list, currency and so on) outside Interactor. Please refactor this.
If you need any external data in DataSource or Sorter - you should fetch it in presenter from interactor and after that provide data to Handlers (Helpers)